src/test/groovy/org/prebid/server/functional/testcontainers/PbsConfig.groovy

```diff
@@ -64,6 +64,7 @@ LIMIT 1
 
     static Map<String, String> getDefaultBiddersConfig() {
         ["adapter-defaults.enabled"                   : "false",
+         "adapter-defaults.ortb-version"              : "2.5",
```

Comment:
I'd say let's go with 2.6 by default.

Reply:
Some tests work only on 2.5 and for them need a special spawned container with 2.6, so I think to leave it as 2.5